src/plugins/plugin-knex.ts

```diff
@@ -18,7 +18,6 @@
 var shimmer = require('shimmer');
 var util = require('util');
 
-// knex 0.10.x and 0.11.x do not need patching
```

Comment:
Maybe it wouldn't hurt to keep this comment near the `versions: '>=0.10 <=0.11'` line below to make it explicit that those versions don't need to be patched.

Reply:
Done